.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: "Ruby on Rails CI"
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
```

Comment:
this will run twice on every branch push  Please don't.
```
push:
  branches: ['main']
pull_request:
  types: ['opened', 'reopened', 'synchronize', 'unlocked']
```

Reply:
Please see https://github.com/rubyforgood/stocks-in-the-future/pull/26/commits/43fcb0c48d22d70ec82b5cfa63a5c5f92879b8b8